src/mount/mount-tool.c

```diff
@@ -1104,27 +1104,40 @@ static int acquire_removable(struct udev_device *d) {
         return 1;
 }
 
+static int discover_loop_backing_file(void) {
+        _cleanup_free_ char *escaped = NULL;
+
+        if (arg_mount_where)
+                return 0;
+
+        escaped = xescape(basename(arg_mount_what), "\\");
```

Comment:
hmm also, you first check above that arg_mount_where is set, and then exit early, hence this code is never executed, is it?